packages/twenty-front/src/modules/object-record/object-filter-dropdown/components/ObjectFilterDropdownTextInput.tsx

```diff
@@ -38,7 +38,7 @@ export const ObjectFilterDropdownTextInput = () => {
   };
 
   return (
-    <DropdownMenuItemsContainer>
+    <DropdownMenuItemsContainer width="auto">
```

Comment:
I'm a bit confused about this:

We don't want both DropdownMenuItemsContainer and Dropdown to have a width. Looking at the code, it seems that the EditableFilterDropdownButton and ViewBarFilterDropdown have a width (to 200px and 208px btw). So this change makes sense.

However, don't we have content with different width: the date picker used in date filter is wider than the available field list for instance. ==> I would expect the Dropdown to let the DropdownContent set its width here

Reply:
This is an ongoing topic that I'd like to tackle, see here : https://github.com/twentyhq/core-team-issues/issues/163

If it's not for next sprint then soon.